server/src/infra/entities/asset.entity.ts

```diff
@@ -149,6 +149,9 @@ export class AssetEntity {
   @OneToMany(() => AssetFaceEntity, (assetFace) => assetFace.asset)
   faces!: AssetFaceEntity[];
 
+  @Column({ type: 'timestamptz', nullable: true })
+  facesRecognizedAt?: Date | null;
```

Comment:
I chose a timestamp instead of a simple boolean here since I could see this being useful for more powerful functionality in the future (like "re-run recognition for all assets without a face since $date" after you tweak some model settings). The cost is ~trivial. But, not attached to this and happy to switch to a boolean.